### ISSUE TITLE
Selected Screen (Current Screen) is intractable

### ISSUE THREAD

**Issue by morteza-bahrami** [RafaelBarbosatec/hidden_drawer_menu#30]
Hello guys,
I've started to use the fabulous Hidden Drawer Menu, but my problem is that, while the menu is opened screen on the right is intractable.
What could I do to prevent it from interacting?
I want it opened if the user tapped on it.

Kind Regards

**Comment by hinterlandsupplyco** [comment]
Wrap your screen in a `IgnorePointer` widget.

```dartlang
IgnorePointer(
  ignoring: isMenuOpen,
  child: // rest of your screen code
);
```

and set the field ``isMenuOpen`` using the `getMenuStateListner()` stream.

**Comment by abhee4** [comment]
> Wrap your screen in a `IgnorePointer` widget.
> 
> ```dart
> IgnorePointer(
>   ignoring: isMenuOpen,
>   child: // rest of your screen code
> );
> ```
> 
> and set the field `isMenuOpen` using the `getMenuStateListner()` stream.

Can you please elaborate a bit with an example please?

**Comment by RafaelBarbosatec** [comment]
functionality available in the version 2.0.0.
